events.html

```diff
@@ -57,21 +57,68 @@
 
   <body>
     <nav>
-      <ul class="navbar-ul">
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="nav-login">
+        <a
+          class="btn-login"
+          href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+        >
+          <button class="btn-login-text">
+            Sign In
+            <img
+              src="img/github.png"
+              alt="GitHub Icon"
+              height="15px"
+              width="15px"
+            />
+          </button>
+        </a>
+        <div class="user-greet">
+          <div class="user-greet-msg">Hello text</div>
+          <img class="user-profile-pic" />
+        </div>
+      </div>
+      <ul class="nav-menu">
```

Comment:
Are you using two different navbars for desktop and mobile view??

Reply:
No, but I am using two login buttons for the same